### ISSUE TITLE
Can make nord villager's damage based on their weapons ?

### ISSUE THREAD

**Issue by MokoBlack** [TheExceptionist/Coherent-Villages#33]
by the test 
i find that villager's defenses based on what armor they wear 
but the damage of them are different  
they can kill zombie by one hit no matter what weapons they hand.(iron diamon.. just the same)
 when i take their weapon out of them , they still kiled the zombie by one punch（Saitama~！）
so.. i think it need to be something change to get it mor interest

**Comment by MokoBlack** [comment]
and nord villager equiped shield  is they can use it to block attack?

**Comment by TheExceptionist** [comment]
Yes, they should be able to block damage, and the Nords don't just do damage based on their weapon. Like the health, it depends on what class the Nord is.

**Comment by TheExceptionist** [comment]
Yea, they should be able to do that. But with any other armored mobs or bandits, it's much more even handed

**Comment by MokoBlack** [comment]
The villagers are too strong at the moment.
There's nothing to beat them. because the powerful class are everywhere 
Maybe you can refer to the ToroQues mod 
At last i just think one of  balanced way is they have a little of base damage and they can incrase damage by using weapon. high level class have more base damage

**Comment by MokoBlack** [comment]
ok.. i just want it can be more reasons for player to help village
They will be more willing to live in villages 
The villagers will help their life and they will also spend more efforts to help the villagers live, like make more good weapon for them and heal the injury ,just help the village's develop

**Comment by MokoBlack** [comment]
and then i have a idea
add some Fugitive their spawn at night they are weaker Neutral mob , will easy killed by other mob ,if  there is a building that fits the standard of the ‘’house‘’(such as bed,roof,door,wall ) near their location  they will go in it 
Player can trade with them to Increase the friendly, trade for weapon will make they stronger 
Thats a remedial measures for villager losed

**Comment by TheExceptionist** [comment]
That may take some time to implement
